shopsafe-frontend/src/app/google-chart/line-chart/line-chart.component.html

```diff
@@ -0,0 +1 @@
+
```

Comment:
Is this a new file that will be added to?

Reply:
This doesn't actually contain anything because all the information is rendered within the component, but we have to have an `.html` file lol